src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -55,15 +54,13 @@ which Flutter runs:
 |Web     |Edge 1.2.0            |
 |Windows |Windows 10            |
 
-Note that Android SDK 20 is covered by
-testing Android SDK 19, as the differences
-between the two platform versions are minimal.
+* Android SDK 20 is covered by testing Android SDK 19. SDK 20 is effectively
+  API 19 with additional support for Android Wear and no new or deprecated API.
```

Comment:
```suggestion
* Passing tests on Android SDK 19 also confers a passing result on SDK 20.
  This is because Android SDK 20 has additional support for Android Wear,
  but otherwise no new or deprecated API.
```